src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLStopImpl.java

```diff
@@ -246,11 +248,15 @@ public DataFetcher<Iterable<TripTimeOnDate>> stopTimesForPattern() {
 
           // TODO: use args.getLegacyGraphQLOmitCanceled()
 
+          Instant startTime = args.getLegacyGraphQLStartTime() != 0
```

Comment:
This piece of code is repeated quite a bit. Can we extract a method?

Reply:
Done